src/com/dakl/Main.java

```diff
@@ -16,9 +16,12 @@ public static void main(String[] args) {
 
         CheckerBoard game = CheckerBoard.blankBoard();
 
+        // REVIEW : Setup method
         Space start_loc = game.getBoardSpace(Main.parseSpace("C4R5"));
         Team r = game.red;
+        // REVIEW : calling static methods
         Main.threeByThree(game.black, new Team[] {r, r, r, r}, start_loc, true, game);
+        // REVIEW : using public members
```

Comment:
These multiple method invocations seem like they belong in a single method somewhere else instead of having all of them be called inside the main method.